src/crime_vancouver_eda.md

```diff
@@ -0,0 +1,55 @@
+# Exploratory data analysis of Crimes in Vancouver data set
+
+# Summary of the data set
+
+The data set used in this project comes from the Vancouver Police Department Open Data (The Vancouver Police Department, 2021) and can be found here (https://geodash.vpd.ca/opendata/). Each row in the data set represents a reported crime in Vancouver, including crime type, date (in year, month, day, hour, minute), block address, and neighbourhood name. The data set covers cases from 2003 to 2021 of a total of 782,749 cases. For the visualization purpose, we focus on the past ten years' data here (from 2020 to 2021).
```

Comment:
... past ten years' data here (from ~2020~ 2011 to 2021)

Reply:
Remember that we drop data in the preprocessing. we only have a couple of years not the entire series